tests/Makefile

```diff
@@ -34,7 +34,15 @@ REGRESSIONS :=  $(shell ls test_cases/)
 all: $(REGRESSIONS)
 
 $(REGRESSIONS):
+ifeq ($(TRAVIS),true)
+	@echo "travs_fold:start:$@"
+endif
 	cd test_cases/$@ && $(MAKE)
+ifeq ($(TRAVIS),true)
+	@echo "travs_fold:end:$@"
```

Comment:
Is there an `i` mising here?

Reply:
thanks. I don't even know how that ever worked then.